cmd/lakectl/cmd/branch_tools_test.go

```diff
@@ -0,0 +1,117 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/treeverse/lakefs/pkg/metastore"
+	"github.com/treeverse/lakefs/pkg/metastore/mock"
+)
+
+const (
+	dbName     = "source"
+	repoName   = "repo"
+	branchName = "branch"
+)
+
+var (
+	validPath   = fmt.Sprintf("s3://%s/%s/path/to/schema", repoName, branchName)
+	invalidPath = fmt.Sprintf("s3/%s/%s/", repoName, branchName)
```

Comment:
Can be added to the constants above just by using '+'. But I really prefer to see these values inside the single func scope it is used.  Using globals, even for test code, without a specifc name - it is something I try to avoid.